src/interfaces/task.type.ts

```diff
@@ -34,14 +34,14 @@ type task = {
 };
 
 enum Tab {
+    IN_PROGRESS = 'IN_PROGRESS',
     ASSIGNED = 'ASSIGNED',
-    COMPLETED = 'COMPLETED',
     AVAILABLE = 'AVAILABLE',
```

Comment:
AVAILABLE = 'UNASSIGNED'

Reply:
actually this is a enum which is getting matched with the backend, we can't change here otherwise we wont be getting data from backend, in Tabs/index.tsx file I have written a function which handles the renaming that part. request you please refer that file

Reply:
for changing this in the backend, before creating an issue ticket  we need to take approval from ankush

Reply:
Please ask and take approval...I think ankush will also say yes